policyengine_us/parameters/gov/states/ga/tax/income/agi/exclusions/military/additional/amount.yaml

```diff
@@ -0,0 +1,14 @@
+description: Georgia provides additional military retirement income exclusion at this amount.
+metadata:
+  label: Georgia additional military retirement income exclusion amount
+  unit: currency-USD
+  reference:
+  - title: 2021 IT-511 Individual Income Tax Booklet - Subtractions(2)
+    href: https://dor.georgia.gov/document/booklet/2021-it-511-individual-income-tax-booklet/download#page=15
```

Comment:
```suggestion
    href: https://dor.georgia.gov/document/booklet/2021-it-511-individual-income-tax-booklet/download#page=16
```

Reply:
use the pages of the pdf when searching and not document page numbers if they divert

Reply:
adjust in all yaml files